src/Pages/BookDetails/BookDetails.js

```diff
@@ -1,2 +1,185 @@
-import React from "react";
-import ReactDOM from "react-dom";
+import React, { useState } from "react";
+import styled from "styled-components";
+import DescSection from "./DescSection";
+import FinishBook from "./FinishBook";
+import CommentSection from "./CommentSection";
+import IntroSection from "./IntroSection";
+import Aside from "./Aside";
+
+function BookDetails() {
+  // const [data, setData] = useState({ book: [] });
+  const [open, setOpen] = useState(false);
```

Comment:
```suggestion
  const [isOpened, setIsOpened] = useState(false);
```
역시 boolean 타입을 표현하기에 적절한 변수명을 골라주는 게 좋겠습니다

Reply:
네!